ivy/functional/backends/tensorflow/activations.py

```diff
@@ -33,10 +32,20 @@ def relu(x: Tensor, /, *, out: Optional[Tensor] = None) -> Tensor:
     return tf.nn.relu(x)
 
 
-def sigmoid(x: Tensor, /, *, out: Optional[Tensor] = None) -> Tensor:
-    if not ivy.is_array(x):
-        x = float(x)
-    return tf.nn.sigmoid(x)
+def sigmoid(
+    x: Tensor, /, *, out: Optional[Tensor] = None, complex_mode="jax"
+) -> Tensor:
+    if x.dtype.is_complex:
+        real_part = tf.math.real(x)
+        imag_part = tf.math.imag(x)
+
+        real_result = 1 / (1 + tf.exp(-real_part))
+        imag_result = 1 / (1 + tf.exp(-imag_part))
+
+        result = tf.complex(real_result, imag_result)
```

Comment:
`tf.nn.sigmoid` already supports complex numbers, so I don't think this change is needed

Reply:
Done